docu/files/ModuleExtensibility.html.md

```diff
@@ -8,14 +8,13 @@ order: 10
 
 After you have implemented, deployed, and published your Builder UI module, it is visible in the Builder for tenants who subscribe to the package containing the module. In other words, the basic functionality of the Builder can be extended with additional modules when tenants subscribe to the related packages.
 
-Now, imagine that subscriptions could extend specific Builder UI modules themselves. Imagine that, when a user subscribes to "package A,"  the Builder "module A" is enabled. Additionally, the subscription triggers the declaration of an extension point in "slot A" of module A so that, when the user subscribes to package B, the extension point enriches module A by including a plug-in for module B in slot A.
+The concept of extensibility applies not only to Builder, but also to Builder modules themselves. It's possible to extend Builder modules by means of plug-ins. For example, imagine a user subscribes to "package A". By doing this, the Builder "module A" is subsequently enabled. Module A contains an extension point called "slot A". The subscription to module A also triggers the declaration of an extension point in "slot A". Now, when the user subscribes to a "package B" that has a plug-in Builder module defined for display in module A, slot A, the extension point of slot A enriches module A with a plug-in from module B.
```

Comment:
The concept of extensibility applies not only to the Builder, but also to the Builder modules themselves. It's possible to extend Builder modules by means of plug-ins. For example, imagine a user subscribes to "package A". This action enables the Builder "module A". Module A contains an extension point called "slot A". The subscription to module A also triggers the declaration of an extension point in "slot A". Now, when the user subscribes to a "package B" that has a plug-in Builder module defined for display in module A, slot A, the extension point of slot A enriches module A with a plug-in from module B.